webapp/src/index.tsx

```diff
@@ -28,18 +30,62 @@ const developmentMode = process.env.NODE_ENV === 'development';
 root.render(
   <React.StrictMode>
     <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<Landing />} />
-        <Route path={USERS_URL} element={<Users />} />
-        <Route path={PLAY_URL} element={<Play />} />
-        <Route path={CHAT_URL} element={<Chat />} />
-        <Route path={USER_URL} element={<UserPage isMe={true} />} />
-        <Route path={`${USERS_URL}/:id`} element={<UserPage isMe={false} />} />
-        <Route path={EDIT_USER_URL} element={<EditUserPage />} />
-        {developmentMode && (
-          <Route path={COMPONENTS_BOOK_URL} element={<ComponentsBook />} />
-        )}
-      </Routes>
+      <AuthProvider>
+        <Routes>
+          <Route path="/" element={<Landing />} />
+          <Route
+            path={USERS_URL}
+            element={
+              <RequireAuth>
+                <Users />
+              </RequireAuth>
+            }
+          />
+          <Route
+            path={PLAY_URL}
+            element={
+              <RequireAuth>
+                <Play />
+              </RequireAuth>
+            }
+          />
+          <Route
+            path={CHAT_URL}
```

Comment:
![Screenshot from 2022-08-22 15-00-49](https://user-images.githubusercontent.com/2774420/185917724-90c331b1-c5db-487b-9eeb-ccac1690190a.png)
There is error in the console when entering chat or play routes.

Reply:
This error was because the `DispatchPage` from `Play` and `Chat` pages had an undefined `genericEndpointUrl`.

I've marked these props as required and copied/pasted the code from the `Users` page until we implement them.

Not sure if it's the best solution, but it works. Please let me know what you think.

Reply:
sorry, this was my fault, then.
Anyway, thank you for fixing it.